qiita_db/handlers/tests/test_user.py

```diff
@@ -34,7 +34,32 @@ def test_get(self):
         # and that one of the key's info is correct
         obs = obs['data']
         exp = {"password": "$2a$12$gnUi8Qg.0tvW243v889BhOBhWLIHyIJjjgaG6dxuRJk"
-               "UM8nXG9Efe", "email": "shared@foo.bar", "level": "user"}
+               "UM8nXG9Efe", "email": "shared@foo.bar", "level": "user",
+               "name": "Shared"}
+        self.assertEqual(obs, exp)
+
+
+class UsersListDBHandlerTests(OauthTestingBase):
+    def test_get_no_header(self):
+        obs = self.get('/qiita_db/users/')
+        self.assertEqual(obs.code, 400)
+
+    def test_get(self):
+        obs = self.get('/qiita_db/users/',
+                       headers=self.header)
+        self.assertEqual(obs.code, 200)
+
+        obs = loads(obs.body)
+        self.assertEqual(obs.keys(), ['data'])
```

Comment:
As far as I understand this test can just check for dictionary equality instead of checking for the keys and then the values i.e. replace line 53 to 62 for something like:

```python
exp = {'data': [....]}
self.assertEqual(obs, exp)
```